net6_bread_be/appsettings.json

```diff
@@ -5,6 +5,8 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "BreadDbConnection": "Host=localhost;Port=5433;Username=postgres;Password=password123;Database=BreadDB"
```

Comment:
Is your local db on pgAdmin called "BreadDB" or "net6_bread_be"?

Reply:
I fixed this, it wasn't pushed up but it is now